lib/jekyll-seo-tag.rb

```diff
@@ -35,8 +35,8 @@ def info
       }
     end
 
-    def template_contents
-      @template_contents ||= File.read(template_path).gsub(/(>\n|[%}]})\s+(<|{[{%])/,'\1\2').chomp
+    def template
+      @template ||= Liquid::Template.parse(File.read(template_path).gsub(/(>\n|[%}]})\s+(<|{[{%])/,'\1\2').chomp)
```

Comment:
Maybe keep `template_contents` to encapsulate the reading, and add `template` to encapsulate the parsing :)